app/controllers/api/v1/bulk_export_controller.rb

```diff
@@ -17,60 +17,55 @@ def index
         else
           exports = @api_user.bulk_exports
         end
-        render json: exports.to_json(except: [:updated_at, :user_id, :collection_id], include: {:collection => {only: [:title, :slug]}})
+        render json: exports.to_json(except: [:updated_at, :user_id, :collection_id], include: { collection: { only: [:title, :slug] } })
       else
         render status: 401, json: 'You must use an API token to access bulk exports'
       end
     end
 
-
     def start
       if @api_user
         collection_slug = params[:collection_slug]
-        if collection_slug
-          collection = Collection.where(slug: collection_slug).first
-          if collection.nil?
-            render status: 404, json: "No collection exists with the slug #{collection_slug}"
-            return
-          end
+        collection = Collection.where(slug: collection_slug).first
+        if collection.nil?
+          render status: 404, json: "No collection exists with the slug #{collection_slug}"
+          return
+        end
 
-          if collection.show_to?(@api_user)
-            # try to parse the bulk_export
-            bulk_export = BulkExport.new
-            bulk_export.collection = collection
-            bulk_export.user = @api_user
-            bulk_export.status = BulkExport::Status::NEW
-            bulk_export.plaintext_verbatim_page = !(params[:plaintext_verbatim_page].blank?)
-            bulk_export.plaintext_verbatim_work = !(params[:plaintext_verbatim_work].blank?)
-            bulk_export.plaintext_emended_page = !(params[:plaintext_emended_page].blank?)
-            bulk_export.plaintext_emended_work = !(params[:plaintext_emended_work].blank?)
-            bulk_export.plaintext_searchable_page = !(params[:plaintext_searchable_page].blank?)
-            bulk_export.plaintext_searchable_work = !(params[:plaintext_searchable_work].blank?)
-            bulk_export.tei_work = !(params[:tei_work].blank?)
-            bulk_export.html_page = !(params[:html_page].blank?)
-            bulk_export.html_work = !(params[:html_work].blank?)
-            bulk_export.subject_csv_collection = !(params[:subject_csv_collection].blank?)
-            bulk_export.subject_details_csv_collection = !(params[:subject_details_csv_collection].blank?)
-            bulk_export.table_csv_collection = !(params[:table_csv_collection].blank?)
-            bulk_export.table_csv_work = !(params[:table_csv_work].blank?)
-            bulk_export.notes_csv = !(params[:notes_csv].blank?)
-            bulk_export.save
-            bulk_export.submit_export_process
+        if collection.show_to?(@api_user)
+          # try to parse the bulk_export
+          bulk_export = BulkExport.new
+          bulk_export.collection = collection
+          bulk_export.user = @api_user
+          bulk_export.status = BulkExport::Status::NEW
+          bulk_export.plaintext_verbatim_page = !params[:plaintext_verbatim_page].blank?
```

Comment:
Can you remind me what the precedence is of `!` versus `.blank?` ?  I'm trying to figure out why we added parentheses here.

Reply:
`blank?` function produces boolean. We just invert the boolean value here.

Ideally I'd like to change this to `.present?` instead so we don't need to invert. But I kept this similar to old implementation as I may have thought this is a preferential thing. Do you want me to change it to `present?`

Reply:
Sure!  I wasn't aware that `present?` existed universally.